src/process.gr

```diff
@@ -197,23 +197,23 @@ provide let this = () => {
 }
 
 @unsafe
-provide let send: (Process, a, Mailbox<a>) => Result<Void, Exception> =
-  (
-    process,
-    data,
-    mailbox,
-  ) => {
+provide let send: (Process, a, Mailbox<a>) => Result<Void, Exception> = (
+  process,
+  data,
+  mailbox,
+) => {
   let payload = Marshal.marshal(data)
   createData(mailbox, Int64.fromNumber(Bytes.length(payload)))
   writeData(payload)
   sendRaw(process)
 }
 
-provide let receive: (?timeout: Uint64, Mailbox<a>) => Result<a, Exception> =
-  (
-    timeout=0xffffffffffffffffuL,
-    mailbox,
-  ) => {
+// TODO(#2081): Remove formatter ignore after formatter fix is merged
```

Comment:
You need to specify that this TODO is in another repository.

Reply:
I removed the `TODO` as the pr was merged. solving that.